plugins/octopus-deploy/src/components/EntityPageOctopusDeploy/EntityPageOctopusDeploy.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { useReleases } from '../../hooks/useReleases';
+import { getAnnotationFromEntity } from '../../utils/getAnnotationFromEntity';
+import React from 'react';
+import { ReleaseTable } from '../ReleaseTable';
+
+export const EntityPageOctopusDeploy = (props: { defaultLimit?: number }) => {
+  const { entity } = useEntity();
+
+  const projectId = getAnnotationFromEntity(entity);
```

Comment:
instead of throwing inside that function, perhaps there should be an `<Alert` here or something that nicely informs the user that the entity didn't have the required annotation?

Reply:
I've added an additional check here for a missing annotation.

Reply:
As it turns out this is a bit tricky to do here, because I cannot use the hooks conditionally. I wonder if it is really a problem though as this will typically be used on the CI/CD page, which already does a check for the annotation.